examples/text-overlay/README.md

```diff
@@ -1,53 +1,3 @@
-# Hello Pipeless!
+# Multi-pose estimation example
```

Comment:
This seems like a copy paste error

Reply:
You're right, I copied it from the pose example. Now I used the same title as in the doc site https://www.pipeless.ai/docs/docs/v0/examples/text-overlay